driver/docker-container/driver.go

```diff
@@ -94,14 +96,14 @@ func (d *Driver) create(ctx context.Context, l progress.SubLogger) error {
 		if err != nil {
 			return err
 		}
-		rc, err := d.DockerAPI.ImageCreate(ctx, imageName, image.CreateOptions{
+		resp, err := d.DockerAPI.ImageCreate(ctx, imageName, image.CreateOptions{
 			RegistryAuth: ra,
 		})
 		if err != nil {
 			return err
 		}
-		_, err = io.Copy(io.Discard, rc)
-		return err
+		defer resp.Close()
+		return jsonmessage.DisplayJSONMessagesToStream(resp, streams.NewOut(io.Discard), nil)
```

Comment:
We can avoid the cli dependency:

```suggestion
		return jsonmessage.DisplayJSONMessagesStream(resp, io.Discard, 0, false, nil)
```